### PR TITLE
End boss < feature

### PR DESCRIPTION
**Pull request by Michieldoesburg**
Ive merged it with master so it can be merged automatically.

### CODE REVIEW

**Comment by SunweiWang** [comment]
Looks really good to me, nicely implemented
< review

**Comment by ClintonCao** [comment]
Looks good to me. Nice job on the spawning of lance and the end boss! < feature

File: src/factories/AnimationTimerFactory.java [hunk 1]
```diff
@@ -48,53 +56,118 @@ public AnimationTimer makeAnimationTimer(GraphicsContext gc) {
...
           Game.getLogger().logSwitchScreen("WinningScreen");
+          
+          MainScreenController.setBossMode(false);
+          playerFish.setHasLance(false);
+          
+          endBossbb.setX(-2000);
+        	endBossbb.setY(-2000);
+        }
+        
+        if (pfbb.getWidth() > 60) {
```
Comment (SunweiWang) [inline src/factories/AnimationTimerFactory.java]: So we just have to change this number, then we can have a longer game, right?

< review

**Comment by MatthijsJWH** [comment]
Seems fine to me, I would like to merge it so that I can check if their are no merge conflicts with my branch.
<review

**Pull request merged**